### PR TITLE
bug-WDP190201-9

### PR DESCRIPTION
**Pull request by Bodeck**
*Task:* Create placeholder with transition properties, to be used for elements with hover
*Changes:*
- create extend.sass file with placeholder
- extend placeholder in all elements which have hover

### CODE REVIEW

File: src/sass/components/_button.scss [hunk 1]
```diff
@@ -15,13 +16,15 @@
 
 .btn-main-small {
   @extend .btn-main;
+  // @extend %hover-transition;
```
Comment (Neonides) [inline src/sass/components/_button.scss]: Jak zakomentowane to wyrzucamy

Comment (Bodeck) [reply src/sass/components/_button.scss]: Done. Usunalem tez line-height, bo jest extendowane z .btn-main i mialo taka sama wartosc.

File: src/sass/components/_post-box.scss [hunk 2]
```diff
@@ -52,7 +53,14 @@
     padding: 25px;
 
     .post-title {
+<<<<<<< HEAD
```
Comment (Neonides) [inline src/sass/components/_post-box.scss]: hmmm?

**Comment by Neonides** [comment]
Konflikty

**Pull request merged**